File: letter/ru/Б.py
from common import *

expr = Eq(
    (x + S(1) / 2) *
    (y - 1) *
    ((x + S(1) / 2) ** 2 + (2 * y + 1) ** 2 - 1) *
    sqrt(
        S((x + S(1) / 2 + eps) * (y - (1 + eps))) /
        ((x - (S(1) / 2 + eps)) * (y + 1 + eps))
    ),
    0
)
size = 1

File: letter/ru/Б.js
import { Letter } from '../../Letter.js';

const expr = '(x + 1/2) * (y - 1) * ((x + 1/2)^2 + (2*y + 1)^2 - 1) * sqrt(((x + 1/2 + 1/10) * (y - (1 + 1/10)))/((x - (1/2 + 1/10)) * (y + 1 + 1/10)))';

export const letter = new Letter(expr, 1);
